### PR TITLE
Add support for mongodb to frontend chart

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

**Review by Jancis — CHANGES_REQUESTED** [review]
Changeset looks very clean and good, this would be a very good addition to the chart. 
The one suggestion from me would be using the latest mongodb chart version as there's a long upgrade path since 5.x, https://github.com/bitnami/charts/blob/master/bitnami/mongodb/Chart.yaml#L29
Seems that the values structure has not changed, but You might still need to confirm it works as expected with the latest version too.

**Comment by badrange** [comment]
I'm not 100% that I understood you, but I hope the latest update is one step on the way. I also added an image tag to the values file, I guess it would be better to default to the most recent version of mongodb. 

Not sure if I got it right and I don't have the setup required to see what happens..

File: charts/frontend/Chart.yaml [hunk 1]
```diff
@@ -15,6 +15,10 @@ dependencies:
   version: 6.17.x
   repository: https://charts.bitnami.com/bitnami
   condition: rabbitmq.enabled
+- name: mongodb
+  version: 10.26.4
```
Comment (Jancis) [inline charts/frontend/Chart.yaml:19]: Set the patch version to "x" so helm dependencies can upgrade to the latest patch version

**Comment by badrange** [comment]
Is there anything left that blocks merging of this PR? 

The potential risk by running this in our project and using us as test bunnies is quite low for two reasons:

* Our use case is running MongoDB as a "query-able cache" of data we fetch from various APIs that can be restored from scratch automatically
* For the next couple of months our project is a non-announced beta site that will only have internal users who don't expect anything to work

**Comment by Jancis** [comment]
The last missing piece was testing this in actual deployment. It's done now, here are few missing pieces that need a fix:
  - chart repo url fix (correct url is https://charts.bitnami.com/bitnami) and lock file needs to be updated (basically `helm dep update charts/frontend && helm dep build charts/frontend` )
  - add mongodb to no_proxy list (see `_helpers.tpl`):
    ```
    - name: no_proxy
      value: .svc.cluster.local,{{ .Release.Name }}-es{{ if $proxy.no_proxy }},{{$proxy.no_proxy}}{{ end }}
    - name: NO_PROXY
      value: .svc.cluster.local,{{ .Release.Name }}-es{{ if $proxy.no_proxy }},{{$proxy.no_proxy}}{{ end }}
    ```
  - Note: branch names can't contain "mongodb" or connection will get screwed up (because of https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_names.tpl#L26-L28).

**Pull request closed without merging**